Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: kinto
 home: https://www.kintohub.com/
-version: 0.2.2
+version: 0.2.3
```

Comment:
better to mark 0.3.0 as this includes some changes rather than just patches

Reply:
agree. done